### PR TITLE
Set pixels brightness; recall display brightness setting in auto dim mode

### PR DESCRIPTION
**Pull request by CedarGroveStudios**
1) Add the capability to set pixel brightness when instantiating PyBadger. Use the standard value range for NeoPixels. If omitted, default to brightness value of 1.0. Example: 
```
from adafruit_pybadger import PyBadger
pybadger = PyBadger(pixels_brightness=0.01)
```
2) Use setter value rather than `brightness=1.0` when restoring to initial brightness with `auto_dim_display()`:
```
pybadger.brightness = 0.9  # set initial display brightness
```

### CODE REVIEW

**Comment by CedarGroveStudios** [comment]
Travis errors are completely Greek to me. I'll need some help with this one.

**Comment by makermelissa** [comment]
Ah, that's an easy one. As it suggests, add 12 spaces to the left of that statement. :)

**Comment by CedarGroveStudios** [comment]
Thanks! Where can I find the Travis details (asked the newbie)?

**Comment by makermelissa** [comment]
On the checks tab.

**Comment by CedarGroveStudios** [comment]
Thanks @makermelissa. The browser app's checks tab didn't have that level of detail. Saw that pylint can be used to reveal the details. May be time to switch to Linux.

File: adafruit_pybadger.py [hunk 1]
```diff
@@ -103,6 +103,7 @@ def __init__(self, i2c=None):
 
         # Display
         self.display = board.DISPLAY
+        self._brightness_value = 1.0
```
Comment (makermelissa) [inline adafruit_pybadger.py]: This appears to be a value for display brightness. Can we name it something like _display_brightness so it doesn't get confused with neopixel brightness? It makes reading the code a bit easier. Thanks

**Review by makermelissa — CHANGES_REQUESTED** [review]

**Comment by makermelissa** [comment]
Sorry, I meant under the Checks tab, you need to click the link labeled "The Build" and it should get you there.

File: adafruit_pybadger.py (hunk 1 above)
Comment (CedarGroveStudios) [reply adafruit_pybadger.py]: Excellent idea. I'll make the change.

**Review by makermelissa — APPROVED** [review]
Looks great! Thanks

**Pull request merged**